c11132674.lua

```diff
@@ -55,7 +55,7 @@ function c11132674.srop(e,tp,eg,ep,ev,re,r,rp)
 	end
 end
 function c11132674.descon(e,tp,eg,ep,ev,re,r,rp)
-	return eg:IsExists(Card.IsLocation,1,nil,LOCATION_ONFIELD+LOCATION_OVERLAY)
```

Comment:
I guess we may just change it into `eg:IsExists(Card.IsLocation,1,nil,LOCATION_ONFIELD)` ?

Reply:
https://github.com/Fluorohydride/ygopro-scripts/pull/1988/commits/b445ab909a1af1320fdcf115b9a99ebd8af0b37a updated.